test/e2e/artifact_test.go

```diff
@@ -245,6 +245,9 @@ var _ = Describe("Podman artifact", func() {
 		// There should be no artifacts in the store
 		rmAll := podmanTest.PodmanExitCleanly("artifact", "ls", "--noheading")
 		Expect(rmAll.OutputToString()).To(BeEmpty())
+
```

Comment:
should we also be checking the reverse case or is it already done?  i guess now we can regress on either.